CHANGELOG-PENDING.md

```diff
@@ -6,12 +6,11 @@ Month, DD, YYYY
 
 ### BREAKING CHANGES
 
-- [go package] (Link to PR) Description @username
-[feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
+- [chore: rename Full node to Bridge node #294](https://github.com/celestiaorg/celestia-node/pull/294) [@Wondertan](https://github.com/Wondertan)
 
 ### FEATURES
 
-- [go package] (Link to PR) Description @username
+- [feat(cmd): give a birth to cel-shed and p2p key utilities #281](https://github.com/celestiaorg/celestia-node/pull/281) [@Wondertan](https://github.com/Wondertan)
```

Comment:
why is this included in this PR?

Reply:
It was placed in the wrong place. Should be fixed in https://github.com/celestiaorg/celestia-node/pull/280 and I will remove the change here while rebasing.

Reply:
👍🏻